maven-scm-api/src/main/java/org/apache/maven/scm/command/changelog/ChangeLogScmRequest.java

```diff
@@ -130,6 +130,18 @@ public void setEndRevision( ScmVersion endRevision )
         parameters.setScmVersion( CommandParameter.END_SCM_VERSION, endRevision );
     }
 
+    public boolean getStartFromRoot()
+        throws ScmException
```

Comment:
4 char indent

Reply:
sorry I do not understand this one.

Reply:
My bad, it is already indented with four chars. All is fine.